### DISCUSSION TITLE
Is this a problem?

### DISCUSSION THREAD

**Discussion by ghost** [seerge/g-helper#1659] [General]
Found this while running windows autorun virustotal check. is it a problem or is it in my laptop only?
![firefox_Y8YM7EQ1cI](https://github.com/seerge/g-helper/assets/76037097/55511d98-30c0-4357-b39c-da0dba199809)

**Comment by seerge** [comment]
> Windows Defender marks app as malware / virus
> False positives from Windows Defender (or any other similar system that uses machine learning for detection) is possible as the application is not digitally signed with a certificate. You can always download a version below or compile the app by yourself. All application sources are open and can be monitored from A to Z :)
